app/models/state_file_ny_intake.rb

```diff
@@ -99,6 +101,8 @@
 class StateFileNyIntake < StateFileBaseIntake
   encrypts :account_number, :routing_number, :raw_direct_file_data
 
+  enum nyc_residency: { unfilled: 0, full_year: 1, part_year: 2, none: 3 }, _prefix: :nyc_residency
+  enum nyc_maintained_home: { unfilled: 0, yes: 1, no: 2 }, _prefix: :nyc_maintained_home
   enum nyc_full_year_resident: { unfilled: 0, yes: 1, no: 2 }, _prefix: :nyc_full_year_resident
```

Comment:
Is there a good convention for marking `nyc_full_year_resident` and `eligibility_part_year_nyc_resident` as deprecated/candidates for future deletion?